rules/windows/other/win_wmi_persistence.yml

```diff
@@ -1,33 +1,52 @@
+action: global
 title: WMI Persistence
 id: 0b7889b4-5577-4521-a60a-3376ee7f9f7b
 status: experimental
-description: Detects suspicious WMI event filter and command line event consumer based on event id 5861 and 5859 (Windows 10, 2012 and higher)
-author: Florian Roth
+description: Detects suspicious WMI event filter and command line event consumer based on WMI and Sysmon logs.
```

Comment:
Since the part with sysmon events was excluded, the description needs to be fixed.

Reply:
Done